PoGo.NecroBot.CLI/Config/Translations/translation.id.json

```diff
@@ -270,7 +270,7 @@
     },
     {
       "Key": "gotUpToDateVersion",
-      "Value": "Sempurna! Kamu punya versi terbaru {0}"
+      "Value": "Info! Silahkan perbarui ke versi terbaru {0}"
```

Comment:
No. This is an information for the user that he got the latest version, please refer [this line.](https://github.com/NECROBOTIO/NecroBot/blob/master/PoGo.NecroBot.Logic/Common/Translations.cs#L325)